src/metabase/models/login_history.clj

```diff
@@ -86,10 +87,8 @@
 (defn- pre-update [_login-history]
   (throw (RuntimeException. (tru "You can''t update a LoginHistory after it has been created."))))
 
-(extend #_{:clj-kondo/ignore [:metabase/disallow-class-or-type-on-model]} (class LoginHistory)
-  models/IModel
-  (merge
-   models/IModelDefaults
-   {:post-select post-select
-    :post-insert post-insert
-    :pre-update  pre-update}))
+(mi/define-methods
+ LoginHistory
+ {:post-select post-select
+  :post-insert post-insert
+  :pre-update  pre-update})
```

Comment:
I added this helper that works (or will work, at least) with Toucan 2 recently and updated everything to use it, but I guess I forgot to update this model.